### DISCUSSION TITLE
Application to chromatin velocity

### DISCUSSION THREAD

**Discussion by timoast** [theislab/scvelo#817] [Q&A] [answered]
In a [recent paper](https://www.nature.com/articles/s41587-021-01031-1), the authors adapted scVelo to study chromatin velocity by replacing unspliced counts with open chromatin counts, and replacing spliced counts with H3K9me3 counts (a repressive histone modification). From my understanding of the scVelo method, it may not make sense to just replace spliced and unspliced counts in this way.

The scVelo model tries to infer a set of parameters (transcriptional rate, transcriptional state, latent time, etc) from the observations of spliced and unspliced counts for each gene using an EM algorithm. There is an assumption that there are two steady-states (constant rate of transcription, and gene turned off), that are associated with a constant ratio of spliced to unspliced counts, and that transcriptional induction is associated with a positive deviation from the steady state expectation, and repression is associated with negative deviation. As the steady on state is reached, the ratio of unspliced/spliced will fall back towards the steady-state rate.

In the context of chromatin states, reaching a steady state would not be associated with any relative increase in a repressive histone mark like H3K9me3, and so applying scVelo in that context would violate assumptions of the model. Can anyone comment on whether this makes any sense?

**Comment by WeilerP** [comment, marked as answer]
@timoast, I had only skimmed the paper (will have a closer look again) when it came out and I cannot comment on the chromain part (not a biologist), but: Everything you are saying is correct and assuming your comment on the chromatin part ist correct (from what I heard other people say it is), I do agree with you. The base assumption to apply the current RNA velocity model is, as you mentioned, that the phase portrait has the characteristic almond shape (or at least part thereof).

**Reply by timoast** [reply]
Got it, thanks!

**Comment by dawe** [comment]
Hello @timoast, many wrote me raising that point. We are working on a better (and more appropriate) model that should replace scvelo for these particular data.
In the meantime we found that the difference between open and closed chromatin as measured in the assay can be used to build a cellrank kernel that nicely recapitulates lineages
